bin/checkGoogleCL

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+P=googlecl
```

Comment:
Rename `P`, it's not very descriptive.  Or just don't use a variable for it and hardcode `googlecl` in the script, since it's not likely to change.